app/Repositories/DigitalPublicationArticleRepository.php

```diff
@@ -37,6 +39,12 @@ public function afterSave($object, $fields)
         GeneratePdf::dispatch($object);
     }
 
+    public function setNewOrder($ids)
+    {
+        ReorderNestedModuleItems::dispatch($this->model, $ids)
+        ->onQueue($this->reorderNestedModuleItemsJobQueue);
+    }
+
```

Comment:
This method is defined as is in the `HandleNesting` trait, so no need to redefine it here.